xapian-applications/omega/omegatest

```diff
@@ -204,28 +204,21 @@ printf '$contains{$cgi{a},$cgi{b}}' > "$TEST_TEMPLATE"
 testcase '6' P=text a=fish b="Hello fish"
 testcase '' P=text a="Example" b="random"
 
+NEWLINE='
+'
+
 # Feature tests for $match.
 printf '$match{$cgi{a},$cgi{b},$cgi{c}}' > "$TEST_TEMPLATE"
 testcase 'true' P=text a="ab*c+" b="ac"
 testcase '' P=text a="acb" b="abc"
 testcase 'true' P=text a="[A-Z]bcD" b="abcd" c="i"
 testcase '' P=text a="[A-Z]bcD" b="abcd"
-testcase 'true' P=text a="^abc$" b="def
-abc
-def" c="m"
-testcase '' P=text a="^abc$" b="def
-abc
-def"
-testcase 'true' P=text a="ab(.)*cd" b="ab
-efgh
-cd" c="s"
-testcase '' P=text a="ab(.)*cd" b="ab
-efgh
-cd"
-testcase 'true' P=text a="    #test_comment1
-			[\sab] #test_comment2  " b="de " c="x"
-testcase '' P=text a="    #test_comment1
-			[\sab] #test_comment2  " b="de "
+testcase 'true' P=text a="^abc$" b="def""$NEWLINE""abc""$NEWLINE""def" c="m"
```

Comment:
You can use `${NEWLINE}` rather than starting and stopping the strings, which should be more readable. This test also works if you drop the `def` from the beginning.